README.md

```diff
@@ -0,0 +1,96 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="sammie.jpg" alt="logo" width="500"  height="auto" />
+  <br/>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [PORTFOLIO] <a name="about-project"></a>
+
+**[PORTFOLIO]** is a git and github training project.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<p>HTML, CSS</p>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **HTML**
+- **CSS**
+- **Linter**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ git clone https://github.com:ogoigbe12/Hello-Microverse.git
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  Run with live server
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="samuel ogoigbe"></a>
+
+- GitHub: (https://github.com/ogoigbe12)
+- Twitter: (https://twitter.com/ogoigbe1)
+- LinkedIn: (https://https://www.linkedin.com/in/samuel-ogoigbe-b1919b205/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **demo project**
+- [ ] **responsive design**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [OPTIONAL] Kindly make sure to create the `LICENCE` and add it to this repo and make sure this link directs user the created `LICENCE` file.